src/components/TopArtists/TopArtists.js

```diff
@@ -9,25 +9,43 @@ const TopArtists = ({topArtists}) => {
     return topTen;
   }, []);
 
+  /****** DON'T RUN BLOCK UNTIL WE'VE FIGURED OUT 'CORS' ISSUE ******/
```

Comment:
Thank you for the note.   Have we all researched the expiration date of our api key?